server2/controllers/db.controller.ts

```diff
@@ -0,0 +1,167 @@
+import {BaseController, IBaseController} from "./base.controller";
+import {ILike}                           from "../../shared/types/Entities/ILike";
+import {IPost}                           from "../../shared/types/Entities/IPost";
+import {IUser}                           from "../../shared/types/Entities/IUser";
+import {Collection, Db, MongoClient}     from "mongodb";
+import {config}                          from "../config/config";
+//MongoDB Constants
+const
+	MongoClinet = require('mongodb').MongoClient(),
+	ObjectID    = require('mongodb').ObjectID;
+
+export interface IDBController extends IBaseController {
+	saveLike(like: ILike): Promise<any>
+
+	savePost(post: IPost): Promise<any>
+
+	saveUser(user: IUser): Promise<any>
+
+	getLikes(): Promise<ILike[]>
+
+	getPosts(): Promise<IPost[]>
+
+	getUsers(): Promise<IUser[]>
+
+
+	deletePost(post_id: string): Promise<any>
+
+
+	unLike(like_id: string)
+
+	getPostLikesAmount(post_id: string): Promise<number>
+
+	getPostLikes(post_id: string): Promise<ILike[]>
+
+	getOneUser(user_id: string): Promise<IUser>
+
+
+	getUserLikes(user_id: string): Promise<ILike[]>
+
+	deletePostLikes(post_id): Promise<any>
+
+
+	close(): Promise<any>
+
+}
+
+export class DBController extends BaseController implements IDBController {
+	private client: MongoClient
+	private db: Db
+	likesCollection: Collection
+	postsCollection: Collection
+	usersCollection: Collection
+
+
+	constructor() {
+		super();
+	}
+
+
+	async init(): Promise<void> {
+		const This = this;
+
+		return new Promise((resolve, reject) => {
+			this.client = new MongoClient(config.URL, {
+				useUnifiedTopology: true
+			})
+
+			this.client.connect(function (err) {
+				if (err) {
+					console.error('Mongo Err', err);
+					return reject()
+				}
+				console.log("Connected successfully to Mongo");
+				This.db = This.client.db(config.dbName);
+
+				This.likesCollection = This.db.collection('likes')
+				This.postsCollection = This.db.collection('posts')
+				This.usersCollection = This.db.collection('users')
+
+				resolve()
+			});
+		})
+	}
+
+	async getLikes(): Promise<ILike[]> {
+		return this.likesCollection.find({}).toArray()
+	}
+
+	async getPosts(): Promise<IPost[]> {
+		return this.postsCollection.find({}).toArray()
+	}
+
+	async getUsers(): Promise<IUser[]> {
+		return this.usersCollection.find({}).toArray()
+	}
+
+	async saveLike(like: ILike): Promise<any> {
+		const newLike = await this.likesCollection.insertOne(like);
+		return newLike.ops
+
+	}
+
+	async savePost(post: IPost): Promise<any> {
+		const newPost = await this.postsCollection.insertOne(post)
+		return newPost.ops
+	}
```

Comment:
whats the .ops this?

Reply:
when i return a value with mongo it gives a lot more information than the object himself without the .ops